src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,60 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  handleChange = (e) => {
+    const { name, value } = e.target;
+
+    this.setState({ [name]: value });
+  }
 
   render() {
+    const { addMovie } = this.props;
+
     return (
-      <form>
-        Put the form here
+      <form onSubmit={(e) => {
+        e.preventDefault();
+        addMovie(this.state);
+        this.setState({
+          title: '',
+          description: '',
+          imgUrl: '',
+          imdbUrl: '',
+          imdbId: '',
+        });
+      }}
+      >
+        {Object.keys(this.state).map(stateField => (
+          <div key={stateField}>
+            <label>
+              {stateField}
+              <input
+                value={this.state[stateField]}
+                name={stateField}
+                placeholder={stateField}
+                onChange={this.handleChange}
+              />
+            </label>
```

Comment:
```suggestion
              <input
                value={this.state[stateField]}
                name={stateField}
                placeholder={stateField}
                required
                onChange={this.handleChange}
              />
            </label>
```

вимагаються усі поля, за виключенням description